cmd/run.go

```diff
@@ -24,6 +28,11 @@ import (
 	"go.temporal.io/sdk/log"
 )
 
+const (
+	summaryListenAddr = "127.7.2.1:0"
```

Comment:
any reason we specify "127.7.2.1" over "localhost"?

Reply:
Down with the 127.0.0.1 hegemony!

Reply:
I didn't even realize you could bind to 127.xxx things other than .0.0.1. Wild.

Reply:
It's a whole /8! 16 million addresses and yet everything wants to fight for 127.0.0.1:8080.

Reply:
Hi @mmcshane, on a fresh laptop (MacOS Ventura 13.4.1) I'm finding that I need to run

```
sudo ifconfig lo0 alias 127.7.2.1
```

otherwise I get

```
listen tcp 127.7.2.1:0: bind: can't assign requested address
```

Just mentioning in case you know why that IP isn't available as a loopback by default / whether we need to do something to stop this being an issue for others newcomers in the future.